__tests__/app.test.js

```diff
@@ -82,3 +83,15 @@ describe("200 GET /api/reviews/:review_id should return the information bound to
 	});
 });
 
+describe("GET /api/reviews", () => {
+	test("should return all of the reviews with their corresponding comment count imported from the comment database ascending by date", () => {
+		return request(app)
+			.get("/api/reviews")
+			.expect(200)
+			.then(({ body }) => {
+				expect(body.reviews).toBeSortedBy("created_at", { descending: true });
+			});
+	});
```

Comment:
we should be testing the length of the reviews array and that each review object has all the correct keys